x-pack/metricbeat/module/sql/query/query_integration_test.go

```diff
@@ -178,10 +183,44 @@ func TestPostgreSQL(t *testing.T) {
 				Assertion: func(t *testing.T, event beat.Event) {
 					value, err := event.GetValue("sql.query")
 					assert.NoError(t, err)
-					require.NotEmpty(t, value.(map[string]interface{}))
+					require.NotEmpty(t, value)
+				},
+			}
+
+			// Need to add this.
+			t.Run("fetch with URL", func(t *testing.T) {
+				testFetch(t, cfg)
+			})
+
+		})
+
+		t.Run("merged mode", func(t *testing.T) {
+			cfg = testFetchConfig{
+				config: config{
+					Driver:         "postgres",
+					Queries:        []query{query{Query: "SELECT blks_hit FROM pg_stat_database limit 1;", ResponseFormat: "table"}, query{Query: "SELECT blks_read FROM pg_stat_database limit 1;", ResponseFormat: "table"}},
```

Comment:
Nit. For readability please split this in multiple lines. Something like this at least:
```suggestion
					Queries:        []query{
						query{Query: "SELECT blks_hit FROM pg_stat_database limit 1;", ResponseFormat: "table"},
						query{Query: "SELECT blks_read FROM pg_stat_database limit 1;", ResponseFormat: "table"},
					},
```

Reply:
Re-formatted.